### DISCUSSION TITLE
Struggling with direction

### DISCUSSION THREAD

**Discussion by SiR-PENt** [Cyfrin/foundry-full-course-cu#1946] [General]
Hi!

I just completed the Foundry f23 course and I think I understand about 90% of what was taught. However, I feel like I know this much but don't know where to apply it. Like, for example, what kind of personal projects can I build to cement the knowledge I've gained, or communities with fellow builders I can join?

**Comment by EngrPips** [comment]
Maybe you can build a crowdfunding app where anyone with a business idea can open a proposal request to ask for money, and people can donate to them so they can execute their business idea. You can ponder this concept more to make it more yours.

**Reply by SiR-PENt** [reply]
Hello @EngrPips. Thank you for your suggestion! 

However, will implementing a DAO for proposal requests be a good approach? Please, do let me know what you think

**Reply by EngrPips** [reply]
Yeah, that would make more sense, So the DAO basically votes on which business to sponsor or not.
